ale/drivers/dawn_drivers.py

```diff
@@ -127,11 +127,11 @@ def ephemeris_start_time(self):
         via a spice call but 193 ms needs to be added to
         account for the CCD being discharged or cleared.
         """
-        if not hasattr(self, '_starting_ephemeris_time'):
+        if not hasattr(self, '_ephemeris_start_time'):
             sclock = self.spacecraft_clock_start_count
-            self._starting_ephemeris_time = spice.scs2e(self.spacecraft_id, sclock)
-            self._starting_ephemeris_time += 193.0 / 1000.0
-        return self._starting_ephemeris_time
+            self._ephemeris_start_time = spice.scs2e(self.spacecraft_id, sclock)
+            self._ephemeris_start_time += 193.0 / 1000.0
```

Comment:
Can we get a doc string about where this number is coming from?

Reply:
Should already be in the doc string